### PR TITLE
2.1.0

### PR DESCRIPTION
**Pull request by sjsrey**
- waiting on doc changes for [Smaup](https://github.com/pysal/esda/pull/58#issuecomment-505697354)
This is now fixed with https://github.com/pysal/esda/pull/62/commits/b7bb49c473ffd7105bb956a19356ec2bfd1282f3

### CODE REVIEW

**Comment by sjsrey** [comment]
I think this is close to ready for a release. Other eyes on this PR would be appreciated.

File: esda/getisord.py [hunk 1]
```diff
@@ -2,9 +2,9 @@
 Getis and Ord G statistic for spatial autocorrelation
 """
 __author__ = "Sergio J. Rey <srey@asu.edu>, Myunghwa Hwang <mhwang4@gmail.com> "
-__all__ = ['G', 'G_Local']
+__all__ = ["G", "G_Local"]
```
Comment (jGaboardi) [inline esda/getisord.py]: Is there a specific reason for swapping out the single quotes for doubles quotes?

**Comment by darribas** [comment]
This looks good to me. The PR are only style changes, correct?

**Comment by sjsrey** [comment]
https://github.com/python/black

Sergio (Serge) Rey
Founding Director, Center for Geospatial Sciences
Professor, School of Public Policy
University of California, Riverside
sergio.rey@ucr.edu

File: esda/getisord.py (hunk 1 above)
Comment (sjsrey) [reply esda/getisord.py]: black formatter

**Comment by ljwolf** [comment]
(it'd be cool if we could make black a commit hook. )

**Comment by jGaboardi** [comment]
@ljwolf here's a [very enthusiastic testimonial for `black`](https://github.com/python/black#testimonials):

>Carl Meyer, Django core developer:
>> At least the name is good.

😆

**Pull request merged**